### DISCUSSION TITLE
skip_full_destinations

### DISCUSSION THREAD

**Discussion by onedpb** [swar/Swar-Chia-Plot-Manager#493] [Q&A]
I found skip_full_destinations in the development branch, but I'm not sure (and cannot read from the code) if it measures the remaining size of a destination job-wise or globally.

I think a lot of us have a larger number of jobs and these plots are ending up on a minor number of destination drives. So some jobs will share the same destination drive. I understood that this functionality calculates the remaining size on a destination drive by the actual free space on the drive minus all incoming plots - but does it take the incoming plots from the specific job only or are all incoming plots (from any job on the same machine) to this destination added up?

**Comment by swar** [comment]
Yes. It predicts plot sizes or any jobs plotting to that destination and the future potential job itself.

Calculates all K-Sizes as well... with some margin of error.

**Comment by onedpb** [comment]
Great!!

Thank you for this project, Swar!
